edx_repo_tools/release/tag_release.py

```diff
@@ -70,12 +87,12 @@ def openedx_release_repos(hub, orgs=None, branches=None):
 
     """
     orgs = orgs or OPENEDX_ORGS
-    repos = {}
-
-    for repo, data in tqdm(iter_openedx_yaml(hub, orgs=orgs, branches=branches), desc='Find repos'):
-        if data.get('openedx-release'):
-            repo = repo.refresh()
-            repos[repo] = data
+    repos = catalog_info_file(hub, orgs=orgs, branches=branches)
```

Comment:
This looks like it will find all repos with a catalog-info.yaml file, and if it finds any, it won't look for repos with openedx.yaml.  Is that what you want?

I thought the idea was the check each repo for a catalog-info.yaml file, and if the file isn't there, check for an openedx.yaml file.

Reply:
Ned, you're right, it should look at each repo individually as you said.  @salman2013 can you fix the code so it does that?

Reply:
@nedbat @feanil my bad i misunderstood the requirements, let me update it and come back with another review.

Reply:
@nedbat @feanil I fixed the code logic as per requirement, could you please take an other look, thanks.